content-repo/extra-docs/articles/microsoft-auth-guide.md

```diff
@@ -77,21 +77,24 @@ To configure a Microsoft integration that uses this authorization flow with a se
 7. Save the instance.
 8. Run the `!<integration command prefix>-generate-login-url` command in the War Room and follow the instructions. For example, for Microsoft Graph User: `!msgraph-user-generate-login-url`.
 
-#### Using National Cloud
-Some of the Cortex XSOAR-Microsoft integrations support the deployment of national clouds through the self-deployed
+### Using National Cloud
```

Comment:
I think you can move the paragraph to be first under "Self Deployed Application"